includes/data-stores/class-wc-admin-reports-downloads-stats-data-store.php

```diff
@@ -12,6 +12,15 @@
  */
 class WC_Admin_Reports_Downloads_Stats_Data_Store extends WC_Admin_Reports_Downloads_Data_Store implements WC_Admin_Reports_Data_Store_Interface {
 
+	/**
+	 * Mapping columns to data type to return correct response types.
+	 *
+	 * @var array
+	 */
+	protected $column_types = array(
+		'download_count'    => 'intval',
```

Comment:
Good call on adding this in 👍 Minor arrow alignment issue, but otherwise good here.